packages/app-admin/src/ui/elements/form/FileManagerElement/FileManagerElementRenderer.tsx

```diff
@@ -75,7 +77,8 @@ export class FileManagerElementRenderer extends UIRenderer<
                                     fileManagerElement: element,
                                     showFileManager,
                                     value,
-                                    onChange
+                                    onChange: (value: FileManagerFileItem | null) =>
+                                        onChange({ id: value?.id, src: value?.src })
```

Comment:
Ah good catch! This is something we added a few releases back (the `FileManagerFileItem` type). But, both `onChange` callbacks are actually wrong. You can call `onChange` with an object OR `null`. So calling it like you did will construct an invalid value, when `value === null`.

It should be `onChange(value ? { id: value.id, src: value.src } : null)`, because we can also unset an image, and in that case, the value will be `null`, and we want to propagate that further to the form.